### ISSUE TITLE
Cloud edit <file> does not work

### ISSUE THREAD

**Issue by cynagen** [SquidDev-CC/cloud-catcher#20]
Running `cloud edit <file>` with the latest version of cloud.lua results in the file being called to execute rather than launch the cloud editor. Switchcraft cloud.lua does not appear to be affected being an older version.

**Comment by SquidDev** [comment]
Were you running using the `wget run` version instead?

**Comment by cynagen** [comment]
I was, yes, players on the server (through Discord bridge) confirmed switching back to their local version fixed the problem.

**Comment by SquidDev** [comment]
Yep - it's trying to run `wget` again, instead of `cloud.lua`. Lovely, not sure how to fix that without removing it entirely.
